db/c.cc

```diff
@@ -1027,6 +1027,10 @@ void rocksdb_iter_seek(rocksdb_iterator_t* iter, const char* k, size_t klen) {
   iter->rep->Seek(Slice(k, klen));
 }
 
+void rocksdb_iter_seek_for_prev(rocksdb_iterator_t* iter, const char* k, size_t klen) {
```

Comment:
Please break the line according to clang-format.
Please run build_tools/format-diff.sh or make format